src/Serializer/Serializer.php

```diff
@@ -64,9 +50,6 @@ public function serialize($object)
         return $this->serializer->serialize($object, 'json', $serializationContext);
```

Comment:
I get a warning from PHPStrom that the `serialize` method only takes 1 parameter but 3 parameters are specified here.

Reply:
Good spot. The docblock for the `$this->serializer` property was incorrect! It used the `SerializerInterface` implemented by the class in this file, while it should have been the JMS `SerializerInterface`
Fixed in be3d26cdbb1eadf5c7224820b5d5b83b7fac7157